CONTRIBUTING.md

```diff
@@ -41,50 +41,110 @@ This saves you having to apply code review feedback repeatedly for each fork.
 
 Running the tests necessary to merge into the repository requires:
 
- * Python 3.11.x, and
- * [PyPy](https://www.pypy.org/) [7.3.19](https://downloads.python.org/pypy/) or later.
+ * Python 3.11.x,
+ * [PyPy](https://www.pypy.org/) [7.3.19](https://downloads.python.org/pypy/) or later
+ * [`uv`](https://docs.astral.sh/uv/) package manager
  * `geth` installed and present in `$PATH`
 
 
+`uv` can be installed via curl (recommended; can self-update):
+
+```bash
+curl -LsSf https://astral.sh/uv/install.sh | sh
+```
+
+or pip (requires Python, can't self-update):
+
+```bash
+pip install uv
+```
+
 `execution-specs` depends on a submodule that contains common tests that are run across all clients, so we need to clone the repo with the --recursive flag. Example:
 ```bash
-$ git clone --recursive https://github.com/ethereum/execution-specs.git
+git clone --recursive https://github.com/ethereum/execution-specs.git
+cd execution-specs
 ```
 
 Or, if you've already cloned the repository, you can fetch the submodules with:
 
 ```bash
-$ git submodule update --init --recursive
+git submodule update --init --recursive
+```
```

Comment:
Ah, we don't use submodules as of #1377; I think we can delete `.gitmodules` entirely now to avoid confusion.

And we should delete this section.

Reply:
Removed this is [c52604c](https://github.com/ethereum/execution-specs/pull/1396/commits/c52604ca91ba1b9ea8a89196ae810e8859095fcb).